plugins/entando-plugin-jpkiebpm/src/main/java/org/entando/entando/plugins/jpkiebpm/apsadmin/portal/specialwidget/BpmFormWidgetAction.java

```diff
@@ -354,36 +379,54 @@ protected void initSharedParameters(final BpmWidgetInfo widgetInfo) {
         if (StringUtils.isNotBlank(kieSourceId) && !"null".equalsIgnoreCase(kieSourceId)) {
             this.setKnowledgeSourcePath(kieSourceId);
         }
-        String listOverrrides = widgetInfo.getConfigDraft().getProperty(KieBpmSystemConstants.WIDGET_INFO_PROP_OVERRIDE_ID);
-        if (StringUtils.isNotBlank(listOverrrides) && !listOverrrides.equals("null")) {
-            String[] idStrings = listOverrrides.split(",");
-            this.setOvrd(new HashSet<Integer>());
-            for (String string : idStrings) {
-                this.getOvrd().add(new Integer(string));
+        this.loadKieFormOverrides(widgetInfo.getId());
+    }
+
+    private void loadKieFormOverrides(int widgetInfoId) throws ApsSystemException {
+        List<KieFormOverride> formOverrides = kieFormOverrideManager.getFormOverrides(widgetInfoId);
+        if (!formOverrides.isEmpty()) {
+            if (overrides == null) {
+                overrides = new ArrayList<>();
+            }
+            for (KieFormOverride formOverride : formOverrides) {
+                overrides.add(getKieFormOverrideVO(formOverride));
             }
         }
     }
 
+    private KieFormOverrideVO getKieFormOverrideVO(KieFormOverride kieFormOverride) {
+        KieFormOverrideVO vo = new KieFormOverrideVO();
+        vo.setId(kieFormOverride.getId());
+        vo.setField(kieFormOverride.getField());
+        vo.setActive(kieFormOverride.isActive());
+        for (IBpmOverride bpmOverride : kieFormOverride.getOverrides().getList()) {
+            if (bpmOverride instanceof DefaultValueOverride) {
+                vo.setDefaultValue(((DefaultValueOverride) bpmOverride).getDefaultValue());
+            } else if (bpmOverride instanceof PlaceHolderOverride) {
+                vo.setPlaceHolderValue(((PlaceHolderOverride) bpmOverride).getPlaceHolder());
+            }
+        }
+        return vo;
+    }
+
     protected void loadWidgetInfo() {
         try {
             String widgetInfoId = this.getWidget().getConfig().getProperty(KieBpmSystemConstants.WIDGET_PARAM_INFO_ID);
             if (StringUtils.isNotBlank(widgetInfoId)) {
                 BpmWidgetInfo widgetInfo = this.getBpmWidgetInfoManager().getBpmWidgetInfo(Integer.valueOf(widgetInfoId));
                 if (null != widgetInfo) {
-
                     this.initSharedParameters(widgetInfo);
-
                 }
             }
-
         } catch (ApsSystemException e) {
             logger.error("Error loading WidgetInfo", e);
         }
     }
 
     public String chooseForm() {
         try {
-            //System.out.println("nothing to do");
+            overrides = new ArrayList<>();
+            // TODO: load existing
```

Comment:
Remove comment or resolve TODO

Reply:
Ok, checked, it is not necessary. If you change the process the old overrides are removed when you click on save. There is nothing to load.